main.go

```diff
@@ -52,7 +52,8 @@ func main() {
 
 	var buildSlugs []string
 	environments := createEnvs(cfg.Environments)
-	for _, wf := range strings.Split(cfg.Workflows, "\n") {
+	trimmed := strings.TrimSpace(cfg.Workflows)
+	for _, wf := range strings.Split(trimmed, "\n") {
```

Comment:
trimmed -> strings.TrimSpace(cfg.Workflows) can go inplace

Reply:
in the first line of the loop please have a `wf = strings.TrimSpace(wf)`
because:
```
workflow\n
workflow \n
workflow
```
can happen